docs/release_notes/pending_release.rst

```diff
@@ -5,6 +5,11 @@ Pending Release Notes
 Updates / New Features
 ----------------------
 
+Documentation
+
+* Added additional examples around using the ``smqtk_core.configuration``
```

Comment:
I think it might be a good idea to be a little more specific here, at least referring which examples were added

Reply:
I added a `: ...` addition. PTAL and let me know how it reads.

Reply:
reads well.